Author: Sebastian Harl <sh@tokkee.org>
Description: Set the path of rrd_filter.px to /usr/lib/collectd/utils/.
Index: collectd/contrib/migrate-3-4.px
===================================================================
--- collectd.orig/contrib/migrate-3-4.px
+++ collectd/contrib/migrate-3-4.px
@@ -167,7 +167,7 @@ for (@Files)
 			my $src_ds = $src_dses->[$i];
 			$dest->{'type_instance'} = $type_instances->[$i];
 			$dest_filename = get_filename ($dest);
-			print "./rrd_filter.px -i '$InDir/$orig_filename' -m '${src_ds}:${dst_ds}' -o '$OutDir/$dest_filename'\n";
+			print "/usr/lib/collectd/utils/rrd_filter.px -i '$InDir/$orig_filename' -m '${src_ds}:${dst_ds}' -o '$OutDir/$dest_filename'\n";
 		}
 	}
 	else
@@ -363,19 +363,19 @@ sub special_disk
 		$OutDirs{$dest_directory} = 1;
 	}
 
-	print "./rrd_filter.px -i '$InDir/$orig_filename' -m 'rmerged:read' -m 'wmerged:write' -o '$OutDir/$dest_filename'\n";
+	print "/usr/lib/collectd/utils/rrd_filter.px -i '$InDir/$orig_filename' -m 'rmerged:read' -m 'wmerged:write' -o '$OutDir/$dest_filename'\n";
 
 	$dest->{'type'} = 'disk_octets';
 	$dest_filename = get_filename ($dest);
-	print "./rrd_filter.px -i '$InDir/$orig_filename' -m 'rbytes:read' -m 'wbytes:write' -o '$OutDir/$dest_filename'\n";
+	print "/usr/lib/collectd/utils/rrd_filter.px -i '$InDir/$orig_filename' -m 'rbytes:read' -m 'wbytes:write' -o '$OutDir/$dest_filename'\n";
 
 	$dest->{'type'} = 'disk_ops';
 	$dest_filename = get_filename ($dest);
-	print "./rrd_filter.px -i '$InDir/$orig_filename' -m 'rcount:read' -m 'wcount:write' -o '$OutDir/$dest_filename'\n";
+	print "/usr/lib/collectd/utils/rrd_filter.px -i '$InDir/$orig_filename' -m 'rcount:read' -m 'wcount:write' -o '$OutDir/$dest_filename'\n";
 
 	$dest->{'type'} = 'disk_time';
 	$dest_filename = get_filename ($dest);
-	print "./rrd_filter.px -i '$InDir/$orig_filename' -m 'rtime:read' -m 'wtime:write' -o '$OutDir/$dest_filename'\n";
+	print "/usr/lib/collectd/utils/rrd_filter.px -i '$InDir/$orig_filename' -m 'rtime:read' -m 'wtime:write' -o '$OutDir/$dest_filename'\n";
 }
 
 sub exit_usage
Index: collectd/contrib/migrate-4-5.px
===================================================================
--- collectd.orig/contrib/migrate-4-5.px
+++ collectd/contrib/migrate-4-5.px
@@ -32,8 +32,8 @@ use Data::Dumper ();
 use File::Basename ('dirname');
 
 our $InDir = '/var/lib/collectd';
-our $RRDtool = 'rrdtool';
-our $RRDFilter = 'rrd_filter.px';
+our $RRDtool = '/usr/bin/rrdtool';
+our $RRDFilter = '/usr/lib/collectd/utils/rrd_filter.px';
 
 our %TypesCounterToDerive = # {{{
 (
